cmd/cored/main.go

```diff
@@ -290,6 +290,19 @@ func main() {
 		opts = append(opts, enableMockHSM(db)...)
 		chainlog.Printf(ctx, "Launching as unconfigured Core.")
 		h = core.RunUnconfigured(ctx, db, sdb, *listenAddr, opts...)
+
+		go func() {
+			ticker := time.Tick(5 * time.Second)
```

Comment:
Since we don't need to worry about trying to keep up with an exact 5-second schedule, it might be a little clearer to use Sleep instead:

```go
for {
	core.CheckConfigMaybeExec(ctx, sdb, *listenAddr)
	time.Sleep(5 * time.Second)
}
```